data.py

```diff
@@ -331,8 +338,8 @@ def _get_paths(self):
         p = Path(self.root, self.split)
         for track_path in tqdm.tqdm(p.iterdir()):
             if track_path.is_dir():
-                input_path = list(track_path.glob(self.input_file))
-                output_path = list(track_path.glob(self.output_file))
+                input_path = sorted(list(track_path.glob(self.input_file)))
```

Comment:
Why is this `sorted` needed? `input_path` should only contain one entry, or?

Reply:
:+1: